x-pack/test/functional/apps/security/user_email.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+import { indexBy } from 'lodash';
+export default function ({ getService, getPageObjects }) {
+
+  const PageObjects = getPageObjects(['security', 'settings', 'common', 'accountSetting']);
+  const config = getService('config');
+  const log = getService('log');
+  const esArchiver = getService('esArchiver');
+
+  describe('useremail', function () {
+    before(async () => {
+      await esArchiver.load('discover');
+      await PageObjects.settings.navigateTo();
+      await PageObjects.security.clickElasticsearchUsers();
+    });
+
+    it('should show the default elastic and kibana users', async function () {
+      const users = indexBy(await PageObjects.security.getElasticsearchUsers(), 'username');
+      log.info('actualUsers = %j', users);
+      log.info('config = %j', config.get('servers.elasticsearch.hostname'));
+      if (config.get('servers.elasticsearch.hostname') === 'localhost') {
+        expect(users.elastic.roles).to.eql(['superuser']);
+        expect(users.elastic.reserved).to.be(true);
+        expect(users.kibana.roles).to.eql(['kibana_system']);
+        expect(users.kibana.reserved).to.be(true);
+      } else {
+        expect(users.anonymous.roles).to.eql(['anonymous']);
+        expect(users.anonymous.reserved).to.be(true);
+      }
+    });
+
```

Comment:
This test is already in `x-pack/test/functional/apps/security/users.js:`.  I think you started from that file and forgot to take this out?